client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,67 @@
+.scoreboard > div.scoreboardIntermediate {
+  border: 2px solid black;
+  box-sizing: border-box;
+  background: #fff;
+  border-width: 2px;
+  border-radius: 8px;
+  justify-content: center;
+  align-items: center;
```

Comment:
`justify-content` and `align-items` have no effect without `display: grid` which you also don't need since there is only a single child so this can be removed.